creatorkit/Glossary.md

```diff
@@ -0,0 +1,19 @@
+![Creator Logo](../creatorlogo.png)
+
+# Glossary
```

Comment:
You may want to add more glossary terms
- Gateway
- FlowM2M (Cloud) server
- FlowM2M website
- PicKit3